File: Python/백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
a = int(input())
aa = list(map(int,input().split()))
b,c =map(int,input().split())
sumt = 0    #티셔츠 묶음의 수
sump = a//c    #펜 묶음의 수
for i in aa:
    sumt+=i//b
    if(i%b!=0):
        sumt+=1
p = a%c    #추가 주문하는 펜 수
print(sumt)
print(sump,p)

File: C#/백준/Bronze/30802. 웰컴 키트/웰컴 키트.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int sumt = 0; int sump = a/bb[1]; int p = a%bb[1];
            for (int i=0; i<aa.Length; i++){
                sumt+=aa[i]/bb[0];
                if(aa[i]%bb[0]!=0){
                    sumt++;
                }
            }
            Console.WriteLine($"{sumt}\n{sump} {p}");
        }
    }
}